apps/sms/js/link_helper.js

```diff
@@ -58,7 +58,7 @@ function checkDomain(domain) {
 }
 
 // defines things that can match right before to be a "safe" link
-var safeStart = /[\s,:;\(>]/;
+var safeStart = /[\s,:;\(>\u0080-\uFFFF]/;
```

Comment:
I may ask stupid question now :) It's not related to this patch, but I tested patch with one of the built-in keyboards and I see there're two types of the following symbols ```()``` and ```（ ）```. I tried like ```(175)63759``` ((175)63759 recognized as phone number) and ```（175）63759``` (63759 recognized as phone number).

So, is only the first pair maybe used for phone numbers in Chinese and my second example isn't common\valid ?